config/v1/types_infrastructure.go

```diff
@@ -172,6 +172,9 @@ type BareMetalPlatformStatus struct {
 	// datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames
 	// to the nodes in the cluster.
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
+
+	// Region is the region of the baremetal machine
```

Comment:
@pkliczewski Can you please put more background why this attribute is needed for Baremetal, so I can better document? Thanks

Reply:
@machacekondra sure. We need this attribute to pass cluster location information so it can be scrapped and reported via telemeter.